subprojects/test-runner/instrumentation-tests/src/gradleTest/kotlin/com/avito/instrumentation/StubConfig.kt

```diff
@@ -65,7 +66,20 @@ internal val kotlinStubConfig = """
   |            }
   |        }
   |    }
-  |}    
+  |    
+  |    environments {
+  |       register<com.avito.instrumentation.configuration.KubernetesViaContext>("k8sContext") {
```

Comment:
It will be simpler for users If they can write
```kotlin
environments {
    kubernetesViaContext("name") { }
    ... etc
}
```

Reply:
looks good, yes